cmd/addaccount.go

```diff
@@ -180,6 +180,11 @@ func (p *AddAccountParams) Validate(ctx ActionCtx) error {
 		return fmt.Errorf("account name is required")
 	}
 
+	if strings.Contains(p.name, "/") || strings.Contains(p.name, "\\") {
```

Comment:
You can use `ContainsAny` here (same for other usages in this PR):
```suggestion
	if strings.ContainsAny(p.name, "/\\")